app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,3 @@
+@import "materialize/components/color";
+$primary-color: color("blue", "lighten-2") !default;
```

Comment:
Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.